spec/lib/healthcheck/government_data_check_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+RSpec.describe Healthcheck::GovernmentDataCheck do
+  describe "#status" do
+    context "when government data is unavailable" do
+      it "returns critical" do
+        gov_data_check = described_class.new
+        expect(gov_data_check.status).to be(:critical)
+      end
+    end
+
+    context "when government data is set to expire" do
+      it "gives a warning" do
+        populate_default_government_bulk_data
+        gov_data_check = described_class.new
+        travel_to(7.hours.from_now) do
+          expect(gov_data_check.status).to be(:warning)
+        end
+      end
+    end
+
+    context "when everything is fine" do
+      it "returns ok" do
+        populate_default_government_bulk_data
+        gov_data_check = described_class.new
+        expect(gov_data_check.status).to be(:ok)
+      end
+    end
+  end
+
+  describe "#details" do
+    let(:gov_data_check) { described_class.new }
+
+    it "displays appropriate messages" do
+      government_repo = BulkData::GovernmentRepository.new
+      gov_data_check.instance_variable_set(:@government_repo, government_repo)
```

Comment:
This seems a quite unusual mocking approach. A more common one would be expect to receive new and return instance double. For example: https://github.com/alphagov/content-publisher/blob/e06613a8a3de96245ec7055e362a6dae6da5775f/spec/services/resync_document_service_spec.rb#L115-L118

However I think you do better continuing the approach you did earlier in this test and using the `travel_to` method

Reply:
that seems like a while ago! I've updated the tests to use `travel_to` do you think it's worth having two tests here or would one do, I'm not sure what I'm gaining by testing for 6 and 15 hours...